src/main/java/com/example/batman/repository/CharacterRepositoryImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.example.batman.repository;
+
+import com.example.batman.model.Character;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+import java.util.Collections;
+import java.util.List;
+
+@Transactional
+@Repository("characterRepository")
+public class CharacterRepositoryImpl implements CharacterRepository {
```

Comment:
Do we need a manual Impl for our repositories if we're using Hibernate? I only see interfaces in closeAPI